database/seeds.js

```diff
@@ -3,30 +3,30 @@ const db = require('../database');
 
 // // generate 100 fake user
 for (let i = 0; i < 100; i++) {
-  let user_name = faker.internet.userName();
-  let user_email = faker.internet.email();
-  let user_password = faker.internet.password();
-  db.query(`INSERT INTO users (name,email,password) VALUES ("${user_name}","${user_email}","${user_password}");`);
+  const userName = faker.internet.userName();
+  const userEmail = faker.internet.email();
+  const userPassword = faker.internet.password();
+  db.query(`INSERT INTO users (name,email,password) VALUES ("${userName}","${userEmail}","${userPassword}");`);
 }
 
 
 // // generate 100 fake restaurants
 for (let i = 0; i < 100; i++) {
-  let restaurant_name = faker.company.companyName();
-  let restaurant_address = faker.address.streetAddress();
-  let restaurant_phone = faker.phone.phoneNumber();
-  db.query(`INSERT INTO restaurants (name,address,phone) VALUES ("${restaurant_name}","${restaurant_address}","${restaurant_phone}");`);
+  const restaurantName = faker.company.companyName();
+  const restaurantAddress = faker.address.streetAddress();
+  const restaurantPhone = faker.phone.phoneNumber();
+  db.query(`INSERT INTO restaurants (name,address,phone) VALUES ("${restaurantName}","${restaurantAddress}","${restaurantPhone}");`);
 }
 
 // generate 100 fake reviews for the first 10 restaurants
```

Comment:
You should delete this comment if you're changing from 100 reviews to 3000 reviews.

Reply:
i will do it right now